Jenkinsfile

```diff
@@ -324,6 +341,47 @@ pipeline {
           // FIXME can't be defined globally due to a bug in Jenkins:
           // https://issues.jenkins-ci.org/browse/JENKINS-49826
           matrixBuilder = { def matrix, int level ->
+            variations.failFast = true
+            variations["slurm"] = {
+              try {
+                node("KNL_PRIO") {
+                  parallel "allocation": {
+                    try {
+                      timeout(time: 2, unit: 'HOURS') {
+                        // Allocate a new job
+                        sh """
+                          salloc --job-name=mardyn-test --nodes=1-2 \
+                            --nodelist=mpp3r03c05s01,mpp3r03c05s02\
```

Comment:
should not contain specific nodes

Reply:
Right, i forgot to remove this. There are three nodes available to interactive jobs. I limited the allocation to the first two because i suspected an error i saw only occurred on the third, but all three are affected after all. That's the "Communication connection failure" i'm catching in line 218. So i will remove this.